README.md

```diff
@@ -27,13 +27,12 @@ Make sure to save your changes before exiting the settings page.
 > [!NOTE]
 > 
 > Granting `Read and write permissions` allows workflows to modify your repository, which can include adding or updating files and code. Always ensure that you trust the workflows you enable with these permissions.
- 
-
-![General Settings](docs/images/Github_Initial_Git_Push_Settings_General.png)
 
 ![Workflow Settings](docs/images/Github_Initial_Git_Push_Settings_Actions.png)
 
-The `GITHUB_TOKEN` permissions can also be configured globally for all jobs in a workflow or individually for each job. This example demonstrates how to set the necessary permissions for the `contents` and `pull-requests` scopes on a job level:
+The `GITHUB_TOKEN` permissions can also be configured globally for all jobs in a workflow or individually for each job.
+
```

Comment:
Why are you putting a paragraph in here?

Reply:
Because they are describing two different things, the first sentence highlights how permissions can be configured, the second is about the example code below.

Reply:
Yes, that makes sense to split the sections. Thank you for the explanation.